LACP Resilience update guide

SlimJ4 opened the pull request:
<!--
Hello 👋
Thank you for submitting a Pull Request.

For Work In Progress Pull Requests, please use the Draft PR feature, see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details. As long as your Pull Request is a draft, the OVHcloud Guides Team will not start proofreading it.

For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments.

DO NOT USE a fork if you are an OVHcloud employee. Instead, please contact the OVHcloud Guides Team so to be granted /write access to this repository.

Feel free to apply labels on your Pull Request, selecting them from the list to your right.

Before submitting a Pull Request, please ensure you have:

- 📖 Read the OVHcloud Documentation readme: https://github.com/ovh/docs/blob/develop/readme.md
- 🇬🇧 Edited (at least) the en-gb version of a guide.
- 🇫🇷 Edited also the fr-fr version if you are a French speaker.
- 🌐 Edited all the files if you're doing a fix or a minor update (this is not mandatory but recommended).
- 👷‍♀️ Created a small PR, if applicable.
- ✅ Tested every step you're documenting.
- 📝 Used descriptive commit messages.
- 📐 Resized images which width exceeds 1400px.
- 📝 Edited or blurred any sensitive/private information from your text and images (IPs, user IDs, private URLs, etc.).
-->

## What type of Pull Request is this?

<!-- Delete the lines which don't apply: -->
- New guide(s)

## Description

This guide explains to OVHcloud clients how the new LACP update affects network connectivity resilience, and how to enable it on their existing infrastructure.

## Mandatory information

<!-- Delete the line which doesn't apply: -->
- This Pull Request should not be merged before : 2025/11/17

SlimJ4 committed:
Guide creation

SlimJ4 committed:
FR guide (first version)

SlimJ4 committed:
EN version + schema

SlimJ4 committed:
Update index.md

SlimJ4 commented:
LAST MINUTE INFO : 
The guide should not be published until the 10th of November (initially, it was to be published ASAP).

Y0Coss commented:
@SlimJ4 , please make sure the 2nd guide from your PR contains also a meta.yaml and that the index contains the 2 created guides. BTW, the second guide should also be available in French.

Jessica41 committed:
first proof guide.en-gb.md

Jessica41 committed:
first proof guide.fr-fr.md

Jessica41 committed:
first proof guide.en-gb.md

Jessica41 committed:
Update guide.en-gb.md

SlimJ4 commented:
> @SlimJ4 , please make sure the 2nd guide from your PR contains also a meta.yaml and that the index contains the 2 created guides. BTW, the second guide should also be available in French.

@Y0Coss , my mistake - this second guide is currently a WIP, and should not have been included in this PR. It already has a dedicated branch, and I think what happened is I mistakenly saved a version of this guide in this branch while switching branches. I will remove it ASAP.

SlimJ4 committed:
Delete guide.en-gb.md

Removed the NIST storage wipe guide mistakenly included in this PR - it should have its own.

Y0Coss committed:
Index fix

Y0Coss committed:
Proofreading - meta.yaml edition - Image size reduction

Y0Coss commented:
> LAST MINUTE INFO : The guide should not be published until the 10th of November (initially, it was to be published ASAP).

@SlimJ4 , is it November 10th or October 10th? (see the date you indicated in the PR description)

SlimJ4 commented:
@Y0Coss, I was definitely not in working shape when I posted this ! Second significant mistake here.
It was indeed November 10th, so 2025/11/10. I've been notified today that this is no longer the case, and this should not be merged before November 17th, so I updated the date (correctly this time).

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md:
```diff
@@ -0,0 +1,56 @@
+
+### Ce qui change
+
+Jusqu’à présent, l’agrégation (LACP) des interfaces réseau était réalisée sur des ports appartenant à la **même carte réseau (NIC)**. Cette configuration assurait déjà une redondance en cas de défaillance d’un switch ToR, mais ne couvrait pas le risque de panne d’une carte réseau.
```
```suggestion
Jusqu’à présent, l’agrégation (LACP) des interfaces réseau était réalisée sur des ports appartenant à la **même carte réseau (NIC)**. Cette configuration assurait déjà une redondance en cas de défaillance d’un switch ToR (Top-of-Rack), mais ne couvrait pas le risque de panne d’une carte réseau.
```

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md:
```diff
@@ -0,0 +1,56 @@
+Désormais :
+
+- Les agrégations logiques seront réparties sur **deux cartes réseau distinctes** (sans modification physique du câblage).
+- Les agrégations existantes ne sont pas modifiées.
```
À mettre en premier élément de la liste et ajouer du gras
```suggestion
- Les agrégations existantes **ne sont pas modifiées**.
```

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md:
Ça fait bizarre "permet disponibilité renforcée" sans article

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md:
```diff
@@ -0,0 +1,56 @@
+---
+
+> [!primary]
+> Ce document concerne un déploiement prévu le **31 octobre 2025**, qui changera la manière dont fonctionne l'aggrégation de liens (LACP) au sein des infrastructures OVHcloud.
```
```suggestion
> Ce document concerne un déploiement prévu le **25 novembre 2025**, qui changera la manière dont fonctionne l'aggrégation de liens (LACP) au sein des infrastructures OVHcloud.
```

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.en-gb.md:
```diff
@@ -0,0 +1,56 @@
+
+> [!primary]
+> 
+> This document is related to a deployment planned for **October 31st, 2025**, which will change the way link aggregation (LACP) functions within OVHcloud infrastructures.
```
```suggestion
> This document is related to a deployment planned for **November 25, 2025**, which will change the way link aggregation (LACP) functions within OVHcloud infrastructures.
```

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/images/resilience_comparison.png:
Please use this transparent SVG, it'll look better
![Architecture schema redesign](https://github.com/user-attachments/assets/622dc967-3439-4f04-8382-341c910083da)

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md:
```diff
@@ -0,0 +1,56 @@
+- Les agrégations logiques seront réparties sur **deux cartes réseau distinctes** (sans modification physique du câblage).
+- Les agrégations existantes ne sont pas modifiées.
+
+![Resilience comparison before/after](images/resilience_comparison.png){.thumbnail}
```
```suggestion
![Comparaison de la résilience avant/après](images/resilience_comparison.png){.thumbnail}
```

sbraz commented on pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.en-gb.md:
```diff
@@ -0,0 +1,56 @@
+- Logical aggregations will be spread across **two distinct network cards** (with no physical cabling modification).
+- Existing aggregations are not modified.
+
+![Resilience comparison before/after](images/resilience_comparison.png){.thumbnail}
```
```suggestion
![Before/after resilience comparison](images/resilience_comparison.png){.thumbnail}
```

SlimJ4 committed:
Update pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md

Co-authored-by: Louis Sautier <sautier.louis@gmail.com>

SlimJ4 committed:
Update pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md

Co-authored-by: Louis Sautier <sautier.louis@gmail.com>

SlimJ4 committed:
Update pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.en-gb.md

Co-authored-by: Louis Sautier <sautier.louis@gmail.com>

SlimJ4 committed:
Update pages/bare_metal_cloud/dedicated_servers/lacp-resilience-scale-hg/guide.fr-fr.md

Co-authored-by: Louis Sautier <sautier.louis@gmail.com>

SlimJ4 committed:
Picture change + Clarity fix

Switched the picture to an svg file, small text edits to improve clarity.

Commit:
lacp-resilience-scale-hg: reword, simplify, add doc links

sbraz commented:
@Y0Coss, @SlimJ4, j'ai fait quelques petits changements. Pouvez-vous faire une relecture et publier la documentation aujourd'hui ou demain ?

On va finalement publier immédiatement car la communication envoyée aux clients contiendra un lien vers la page de documentation.

On pourra toujours mettre à jour cette page par la suite si nécessaire.

Y0Coss committed:
Proofreading

Y0Coss commented:
Thank you for your contribution @SlimJ4 and @sbraz
